package pbarray;

message MetaHead {
  repeated int32 items = 1;
}

message DataHead {
  repeated int32 items = 1;
}

message MetaItem {
  optional string filename = 1;
  optional int32 width = 2;
  optional int32 height = 3;
  optional int32 label_id = 4;
  repeated Box boxes = 5;
  optional string label_name = 6;
}

message DataItem {
  optional bytes jpeg = 1;
}

message Box {
  optional int32 start_x = 1;
  optional int32 start_y = 2;
  optional int32 end_x = 3;
  optional int32 end_y = 4;
  optional int32 label = 5;
}
